Dockerfile

```diff
@@ -1,28 +1,31 @@
-# Dockerfile for socorro-collector container
-FROM centos:centos7
+FROM python:2.7.12-slim
 
-# Set Python-related environment variables to reduce annoying-ness
-ENV PYTHONUNBUFFERED 1
-ENV PYTHONDONTWRITEBYTECODE 1
+WORKDIR /app/
+RUN groupadd --gid 1001 app && useradd -g app --uid 1001 --shell /usr/sbin/nologin app
+
+RUN apt-get update && apt-get install -y \
+    gcc apt-transport-https
 
-EXPOSE 8000
+COPY ./requirements.txt /app/requirements.txt
+COPY requirements-dev.txt /app/requirements-dev.txt
 
-# Install pip and get pip8
-RUN yum -y install epel-release \
-    && yum -y update \
-    && yum -y group install "Development Tools" \
-    && yum -y install python-devel python-pip
-COPY bin/pipstrap.py bin/pipstrap.py
-RUN ./bin/pipstrap.py
+RUN pip install -U 'pip>=8' && \
+    pip install --upgrade --no-cache-dir -r requirements.txt
```

Comment:
Looks like we're copying both the requirements files, but we're only installing the production one. That's fine for non-dev environments, but for development, we need the requirements-dev.txt one, too.

How should that work? Can we install dev requirements "only if" the container is for dev purposes?